src/themes/bootstrap3.js

```diff
@@ -44,9 +44,8 @@ JSONEditor.defaults.themes.bootstrap3 = JSONEditor.AbstractTheme.extend({
       group.appendChild(label);
       input.style.position = 'relative';
       input.style.cssFloat = 'left';
-    } 
-    else {
-      group.className += ' form-group';
+    } else {
+      group.className += 'form-group';
```

Comment:
Can you revert this file? It's just formatting and doesn't seem relevant to the pull request.